packages/manager/modules/nasha/src/components/task-tracker/task-tracker.template.html

```diff
@@ -0,0 +1,22 @@
+<task-tracker
+    data-endpoint="{{ $ctrl.endpoint }}"
+    data-heading="{{:: ('nasha_components_task_tracker_operation_' + $ctrl.operation) | translate }}"
+    data-on-done="$ctrl.onDone($success, $error)"
```

Comment:
are we still need the `$` char ?

related to:
https://github.com/ovh/manager/pull/7447#discussion_r913465697